client/internal/bootstrap/csr_test.go

```diff
@@ -4,13 +4,85 @@
 package bootstrap
 
 import (
+	"testing"
+	"strings"
 	"crypto/x509"
 	"encoding/pem"
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	
 )
 
+
+
+func TestMakeKubeletClientCSR(t *testing.T) {
```

Comment:
overall this looks good, my main feedback would be that be we can use `testify`'s assert functionality to make this all a bit less verbose - check out this example: https://github.com/Azure/aks-secure-tls-bootstrap/blob/main/client/internal/bootstrap/event_test.go